packages/simcore-sdk/src/simcore_sdk/nodeports/io.py

```diff
@@ -0,0 +1,86 @@
+import os
+import logging
+
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.orm import exc
+from sqlalchemy.orm.attributes import flag_modified
+
+from simcore_sdk.config.db import Config as db_config
+from simcore_sdk.models.pipeline_models import ComputationalTask as NodeModel
+from simcore_sdk.nodeports import serialization
+
+_LOGGER = logging.getLogger(__name__)
+
+class DbSettings(object):
+    def __init__(self):
+        self._db_config = db_config()
+        self.db = create_engine(self._db_config.endpoint, client_encoding='utf8')
+        self.Session = sessionmaker(self.db)

```

Comment:
Do we really need to keep a reference of the session maker, i.e. ``self.Session``?? If so perhaps we could rename it to ``self.session_maker?``